data/borgmatic.d/config.yaml

```diff
@@ -4,31 +4,41 @@ repositories:
     - path: /mnt/borg-repository
 one_file_system: true
 
-
-#   Passphrase is set in variable $BORG_PASSPHRASE
-#   encryption_passphrase: "DoNotMissToChangeYourPassphrase"
+#   Passphase is set in varibable $BORG_PASSPHRASE
+#   encryption_passphrase: "DonNotMissToChangeYourPassphrase"
```

Comment:
Typo here: "Don"

Reply:
Resolved in https://github.com/borgmatic-collective/docker-borgmatic/pull/303/commits/43a5d3d3aec0eedd9dfef22cb436d9fdc96bb3e5